source/dagonBackend.d

```diff
@@ -1370,17 +1374,32 @@ final class SacScene: Scene{
 	void observerControl(Duration dt){
 		Vector3f forward = fpview.camera.worldTrans.forward;
 		Vector3f right = fpview.camera.worldTrans.right;
+		Vector3f down = fpview.camera.worldTrans.up;
+		Vector3f cameraForward = fpview.camera.observerTrans.forward;
+		Vector3f cameraRight = fpview.camera.observerTrans.right;
+		Vector3f cameraDown = fpview.camera.observerTrans.up;
```

Comment:
I think this means you changed dagon. However, there is no submodule update in this pull request.

Reply:
https://github.com/tg-2/dagon/pull/1

Reply:
is this not how i do this? i just entered the folder, make a commit, and pushed. and it ended up in my dagon repo, so i jsut made a PR against yours

Reply:
I merged the dagon pull request so that you get the correct git commit hash in your update.